src/clanhr_api/core.clj

```diff
@@ -27,9 +27,17 @@
                             (:http-opts opts))}
          opts))
 
+(defn throws-an-error-about-request-time
+  [data response]
+  (when (and (:request-time response)
+             (< 400 (:request-time response)))
```

Comment:
Hum... I'd bump this up to 500. And maybe make it configurable via env. Because if we do that we can turn this on and off. There are several slower requests that maybe we don't want to log to bugsnag (for example, excel downloads).